rosys/vision/simulated_camera/simulated_camera.py

```diff
@@ -37,18 +35,13 @@ def __init__(self,
                                  min_value=1, max_value=30, step=1, default_value=fps)
 
     def to_dict(self) -> dict:
-        return {
-            'id': self.id,
+        return super().to_dict() | {
```

Comment:
Don't we need to put the ID in the dictionary as well?

Reply:
Camera does that already